app/domain/authentication/authn_oidc/authenticate.rb

```diff
@@ -57,14 +60,14 @@ def add_username_to_input
       end
```

Comment:
I think you missed adding `update` in `def_delegators :@authenticator_input, :service_id, :authenticator_name, :account, :username, :request, :webservice`

Reply:
I didn't add it on purpose. The reason we use `def_delegators` is for readability. Someone who reads this class will know easily what are the inputs for this CommandClass, without the need to dig into `authenticator_input`. adding a delegator for `update` will not improve readability as a reader will not understand the usage of this method:
```
def add_username_to_input
      @authenticator_input = @authenticator_input.update(username: conjur_username)
end
```
 if it will be written as:
```
def add_username_to_input
      @authenticator_input = update(username: conjur_username)
end
```

So it's better to leave this method non-delegated and call ` @authenticator_input` explicitly.